src/scenes/Projects/List/ProjectList.tsx

```diff
@@ -56,7 +56,8 @@ export const ProjectList = () => {
       input: {
         ...sort.value,
         filter: {
-          ...omit(filters, 'tab'),
+          type: filters.type ? [filters.type] : undefined,
+          ...omit(filters, 'tab', 'type'),
```

Comment:
What's this about?

Reply:
You made the `type` an array on the API and that broke the filter as it's expecting an array. To keep the radio/single select functionality, we're sending the type value wrapped in a list

Reply:
Let's just change these two spots to match the new API instead
https://github.com/SeedCompany/cord-field/blob/22709b16472c1fc73043dcaacdf4915aa879584d/src/scenes/Projects/List/ProjectFilterOptions.tsx#L22-L22
https://github.com/SeedCompany/cord-field/blob/22709b16472c1fc73043dcaacdf4915aa879584d/src/scenes/Projects/List/ProjectFilterOptions.tsx#L47-L48